tests/unit/hive_metastore/test_migrate_acls.py

```diff
@@ -0,0 +1,198 @@
+    rows = {
+        'SELECT \\* FROM hive_metastore.inventory_database.grants': GRANTS[
+            ("workspace_group", "SELECT", "", "db1_src", "managed_dbfs", ""),
+            ("workspace_group", "MODIFY", "", "db1_src", "managed_mnt", ""),
+            ("workspace_group", "OWN", "", "db1_src", "managed_other", ""),
+            ("workspace_group", "INVALID", "", "db1_src", "managed_other", ""),
+            ("workspace_group", "SELECT", "", "db1_src", "src_view", ""),
+            ("workspace_group", "SELECT", "", "db1_random", "src_view", ""),
+        ],
+        r"SYNC .*": MockBackend.rows("status_code", "description")[("SUCCESS", "test")],
+        'SELECT \\* FROM hive_metastore.inventory_database.groups': GROUPS[
+            ("11", "workspace_group", "account group", "temp", "", "", "", ""),
+        ],
+        "SHOW CREATE TABLE": [
+            {
+                "createtab_stmt": "CREATE OR REPLACE VIEW "
+                "hive_metastore.db1_src.view_src AS SELECT * FROM db1_src.managed_dbfs"
+            }
+        ],
+        'SELECT \\* FROM hive_metastore.inventory_database.tables': UCX_TABLES[
+            ("hive_metastore", "db1_src", "managed_dbfs", "table", "DELTA", "/foo/bar/test", None),
+            ("hive_metastore", "db1_src", "managed_mnt", "table", "DELTA", "/foo/bar/test", None),
+            ("hive_metastore", "db1_src", "managed_other", "table", "DELTA", "/foo/bar/test", None),
+            ("hive_metastore", "db1_src", "src_view", "table", "DELTA", "/foo/bar/test", "select * from foo.bar"),
+        ],
+    }
+    backend = MockBackend(fails_on_first=errors, rows=rows)
+    table_crawler = TablesCrawler(backend, "inventory_database")
+    udf_crawler = UdfsCrawler(backend, "inventory_database")
+    grant_crawler = GrantsCrawler(table_crawler, udf_crawler)
+    group_manager = GroupManager(backend, ws, "inventory_database")
+    workspace_info = ws_info
+    migration_status_refresher = create_autospec(MigrationStatusRefresher)
+
+    principal_grants = create_autospec(PrincipalACL)
+    acl_migrate = ACLMigrator(
+        table_crawler,
+        grant_crawler,
+        workspace_info,
+        backend,
+        group_manager,
+        migration_status_refresher,
+        principal_grants,
+    )
+    migration_status_refresher.get_seen_tables.return_value = {
+        "ucx_default.db1_dst.managed_dbfs": "hive_metastore.db1_src.managed_dbfs",
+        "ucx_default.db1_dst.managed_mnt": "hive_metastore.db1_src.managed_mnt",
+        "ucx_default.db1_dst.managed_other": "hive_metastore.db1_src.managed_other",
+        "ucx_default.db1_dst.dst_view": "hive_metastore.db1_src.src_view",
+    }
+    acl_migrate.migrate_acls()
+    principal_grants.get_interactive_cluster_grants.assert_called()
+
+    assert "GRANT SELECT ON TABLE ucx_default.db1_dst.managed_dbfs TO `account group`" in backend.queries
```

Comment:
`make fmt` complains about copy-paste. run it locally and fix.

```
************* Module tests.integration.account.test_account
tests/integration/account/test_account.py:1:0: R0801: Similar lines in 2 files
==tests.unit.hive_metastore.test_migrate_acls:[98:108]
==tests.unit.hive_metastore.test_table_migrate:[1053:1063]
    principal_grants.get_interactive_cluster_grants.assert_called()

    assert "GRANT SELECT ON TABLE ucx_default.db1_dst.managed_dbfs TO `account group`" in backend.queries
    assert "GRANT MODIFY ON TABLE ucx_default.db1_dst.managed_dbfs TO `account group`" not in backend.queries
    assert "ALTER TABLE ucx_default.db1_dst.managed_dbfs OWNER TO `account group`" not in backend.queries
    assert "GRANT MODIFY ON TABLE ucx_default.db1_dst.managed_mnt TO `account group`" in backend.queries
    assert "GRANT SELECT ON TABLE ucx_default.db1_dst.managed_mnt TO `account group`" not in backend.queries
    assert "ALTER TABLE ucx_default.db1_dst.managed_other OWNER TO `account group`" in backend.queries
    assert "GRANT SELECT ON TABLE ucx_default.db1_dst.managed_other TO `account group`" not in backend.queries
    assert "GRANT MODIFY ON TABLE ucx_default.db1_dst.managed_other TO `account group`" not in backend.queries (duplicate-code)
tests/integration/account/test_account.py:1:0: R0801: Similar lines in 2 files
==tests.unit.hive_metastore.test_migrate_acls:[[41](https://github.com/databrickslabs/ucx/actions/runs/10409941499/job/28830662609?pr=2284#step:5:42):53]
==tests.unit.hive_metastore.test_table_migrate:[990:1002]
GRANTS = MockBackend.rows("principal", "action_type", "catalog", "database", "table", "view")


def test_migrate_acls_should_produce_proper_queries(ws, caplog):
    # all grants succeed except for one
    errors = {"GRANT SELECT ON VIEW ucx_default.db1_dst.view_dst TO `account group`": "TABLE_OR_VIEW_NOT_FOUND: error"}
    rows = {
        'SELECT \\* FROM hive_metastore.inventory_database.grants': GRANTS[
            ("workspace_group", "SELECT", "", "db1_src", "managed_dbfs", ""),
            ("workspace_group", "MODIFY", "", "db1_src", "managed_mnt", ""),
            ("workspace_group", "OWN", "", "db1_src", "managed_other", ""),
            ("workspace_group", "INVALID", "", "db1_src", "managed_other", ""), (duplicate-code)
```

Reply:
Fixed